acceptance/analyzer_test.go

```diff
@@ -158,23 +158,31 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 
-	when("layers path and analyzed path are provided", func() {
-		it("writes analyzed.toml at the provided path", func() {
+	when("analyzed path is provided", func() {
+		it("writes analyzed.toml at the provided path and uses group.toml from the working directory", func() {
```

Comment:
I found the inclusion of "and uses group.toml from the working directory" given that we aren't in one of the "Platform API ..." contexts. Do we even need to mention `group.toml` here, whether we use the given `group.toml` is from the `layers` or working directories given that they are the same in this case?

Reply:
Actually you're right. It's a leftover from the test I changed but we don't really need to test `group.toml` in this scenario.
I'll change it.